_sass/custom/_blog.scss

```diff
@@ -49,4 +49,22 @@ $post-max-width: 1024px;
     text-decoration: underline;
     font-size: 18px;
   }
+
+  img {
+    padding: 25px;
+  }
+
+  blockquote {
+    padding: 2rem 0rem;
```

Comment:
Not necessary to use units for '0' value, so it should just be: padding: 2rem 0;

Reply:
Thank you for a comment! I made all necessary fixes :)